packages/flutter_tools/lib/src/commands/build_aar.dart

```diff
@@ -102,6 +123,14 @@ class BuildAarCommand extends BuildSubCommand {
       'This command assumes that the entrypoint is "lib/main.dart". '
       'This cannot currently be configured.';
 
+  @override
+  Future<void> validateCommand() async {
```

Comment:
same question, is this required for usage values?

Reply:
It is not required for this PR

Reply:
Well turns out this was necessary for this [test](https://github.com/eliasyishak/flutter/blob/cp-migrate-usage-values-ga4/packages/flutter_tools/test/commands.shard/hermetic/build_aar_test.dart#L70-L73) 🙃

Reply:
Let's delete the test rather than change the behavior of the tool in CP.